### PR TITLE
Update README and version

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

Also minor amendment to show info when the image of the VM is a volume.

---

**Related issue :**

### CODE REVIEW

File: fedcloud_vm_monitoring/site_monitor.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def get_vm_image_server_show(self, vm_id):
             return "image name not found"
 
     def get_vm_image(self, vm_id, image_name, image_id):
-        if len(image_name) > 0:
+        if (len(image_name) > 0) and ("booted from volume" not in image_name):
```
Comment (enolfc) [inline fedcloud_vm_monitoring/site_monitor.py]: Maybe what you could do is to use `image_name` as last resort option so first the code inside the `else`

Comment (enolfc) [reply fedcloud_vm_monitoring/site_monitor.py]: so first try to get image, then volume and last use image name (even if it's `booted from volume`)

Comment (sebastian-luna-valero) [reply fedcloud_vm_monitoring/site_monitor.py]: Done, please have a look

**Review by enolfc — APPROVED** [review]
I got a bit lost with the structure, but I trust this works 😉

**Pull request merged**